Products/ZenModel/migrate/updateZentrapConfigs.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import os
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class UpdateZentrapConfigs(Migrate.Step):
+    "Add zentrap.filter.conf and alter zentrap.conf."
+
+    version = Migrate.Version(5,0,70)
+
+    def cutover(self, dmd):
+        ctx = sm.ServiceContext()
+
+        zentrap = filter(lambda s: s.name == "zentrap", ctx.services)[0]
```

Comment:
Ah, no. I was rather simple-mindedly assuming a single collector. I'll perform this operation on all services named zentrap.